pgjdbc/src/main/java/org/postgresql/util/HostSpec.java

```diff
@@ -9,6 +9,7 @@
 
 import org.checkerframework.checker.nullness.qual.Nullable;
 
+import java.util.Objects;
```

Comment:
depending on whether we backpatch this or not, Objects was not available in 1.6

Reply:
I am unable to change the code without having regressions (see: https://github.com/pgjdbc/pgjdbc/pull/2082#issuecomment-791242400)

In addition, other pieces of code in the existing repository must also be modified to ensure compatibility with Java6.
For example:
- pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java
- pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java